.eslintrc.js

```diff
@@ -20,6 +20,14 @@ module.exports = {
   },
   plugins: ["jest", "react", "flowtype", "react-native"],
   rules: {
+    "import/no-extraneous-dependencies": [
```

Comment:
This will most likely not be needed once we merge the #184 PR

Reply:
I will wait for that then.